layouts/partials/main/pages.html

```diff
@@ -3,13 +3,13 @@
     {{ range . }}
         <li>
           <a class="docs-link rounded" href="{{ .Permalink }}">{{ .Title }}</a>
-          {{ template "dokschildren" dict "Page" . }}
+          {{ template "dokschildrenmain" dict "Page" . }}
         </li>
     {{ end }}
 </ul>
 {{ end }}
 
-{{ define "dokschildren" }}
+{{ define "dokschildrenmain" }}
```

Comment:
Differentiating this template from a similarly named one elsewhere.

This template intentionally does not support suppressing pages from this table of contents.